_articles/upgrade-pop.md

```diff
@@ -95,28 +91,39 @@ Once restarted, the computer will be on the newly upgraded system! If you run in
 
 ### Upgrading older releases
 
-Upgrading Pop!\_OS 17.10, 18.10 or 19.04.
+Upgrading Pop!\_OS 18.10 or 19.04.
 
-These older Pop!\_OS releases are now unsupported and no new updates are available. After unsupported versions have been removed from the archive and mirror network, you will need to change where your system checks for un-applied updates to be able to upgrade. Type the following commands in a terminal to switch where `apt` checks for updates. If you would like to have the recovery partition, you will need to backup data and reinstall with a 18.04 or newer release of Pop!\_OS
-
-Pop!\_OS 17.10 is now unsupported and no new updates are available. After unsupported versions have been removed from the archive and mirror network, you will need to change where your system checks for updates to be able to upgrade. Type the following commands in a terminal to switch where `apt` checks for updates. If you would like to have the recovery partition, you will need to backup data and reinstall with a 18.04 or newer release of Pop!\_OS
+These older Pop!\_OS releases are now unsupported and no new updates are available. After unsupported versions have been removed from the archive and mirror network, you will need to change where your system checks for un-applied updates to be able to upgrade. Open a terminal and follow the next set of instructions to upgrade from Pop!\_OS 18.10 or 19.04.
 
+1) Get your current system fully updated:
 ```
-sudo sed -e 's/us.archive./old-releases./' /etc/apt/sources.list > ~/sources.list
-sudo mv ~/sources.list /etc/apt/sources.list
-sudo apt clean
+sudo sed -i 's/us.archive/old-releases/g' /etc/apt/sources.list
 sudo apt update -m
+sudo dpkg --configure -a
+sudo apt install -f
 sudo apt full-upgrade
 sudo apt install pop-desktop
 ```
-For Pop!_OS 18.10 and 19.04 Run
+
+2) Move any PPA additions out of the way, and get all of the sources pointed at the 20.04 versions:
 ```
-sudo pop-upgrade release upgrade systemd
+sudo mkdir -p /etc/apt/sources.list.d/backup
+sudo mv /etc/apt/sources.list.d/* /etc/apt/sources.list.d/backup
+sudo apt-add-repository -yn ppa:system76-dev/stable
+sudo apt-add-repository -yn ppa:system76/pop
+sudo sed -i 's/old-releases/us.archive/g' /etc/apt/sources.list
+sudo sed -i 's/disco/focal/g' /etc/apt/sources.list /etc/apt/sources.list.d/*.list
 ```
-For Pop!_OS 17.10 Run
+
+3) Now,do the upgrade! This will have a three phases and once the download is complete, you can't change your mind. This works from a new install of 19.04, so make sure you have a backup of important data before moving forward. There may be a question about restarting services, and it is safe to answer "Yes". There may be some questions about using "maintainer" version of configuration files and using the that new version is also very likely what you want to do (and the default answer will work):
```

Comment:
"This works from a new install of 19.04, so make sure you have a backup of important data before moving forward."
This is a little bit unclear. Maybe add a part in the middle:
"This works from a new install of 19.04, but some packages may add complications, so make sure you have a backup of important data before moving forward."